### PR TITLE
Curiefense

### PR DESCRIPTION
**Pull request by JuanPabloGarciaMonzon**
In this new PR I have made only 2 commits as the contribution rules demands, 1 with the "make prepare" , "make patch"  and "make clean" commands, in the package/curiefense there is only "generated_changes" and "pachage.yaml".

In the second one  the "make charts" and "make clean" commands have been runned. 

Also the "make validate" command has been runned in my local machine and it has deployed a successful message, specfically . "Successfully validated against migrate_to_v0.2.1_live!"

### CODE REVIEW

File: charts/curiefense/curiefense/1.0.1+up1.0.0/questions.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+questions:
+  - variable: secret.file
+    description: "You need to have access to AWS Credentials in order to create the file s3cfg.yaml in the istio-helm folder'."
+    required: true
+    type: string
+    default: cluster-configuration-secret
+    label: Cluster Configuration
+    group: "Cluster Configuration"
+
+questions:
```
Comment (PennyScissors) [inline charts/curiefense/curiefense/1.0.1+up1.0.0/questions.yaml:10]: There shouldn't be 2 `questions:` keys defined in this file.

File: charts/curiefense/curiefense/1.0.1+up1.0.0/questions.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  - variable: aws.credentials.conversion
+    description: "You need to convert your AWS credentials with these commands cat << EOF | base64 -w0 [default] access_key = xx secret_key = xx EOF"
+    required: true
+    type: string
+    default: cluster-configuration-secret
+    label: Cluster Configuration
+    group: "Cluster Configuration"
+
+  - variable: helm.version
```
Comment (PennyScissors) [inline charts/curiefense/curiefense/1.0.1+up1.0.0/questions.yaml:19]: I see some of the values being set by these questions are not in the chart's values.yaml, which is fine if intentional, but I don't see them being used in any of the templates either (For example this value `helm.version`).

File: charts/curiefense/curiefense/1.0.1+up1.0.0/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+annotations:
+  catalog.cattle.io/certified: partner
+  catalog.cattle.io/display-name: curiefense
+  catalog.cattle.io/release-name: curiefense
+apiVersion: v1
+appVersion: 1.0.0
+description: Complete curiefense deployment
+kubeVersion: '>= 1.19'
+name: curiefense
+version: 1.0.1+up1.0.0
```
Comment (PennyScissors) [inline charts/curiefense/curiefense/1.0.1+up1.0.0/Chart.yaml:10]: This chart is missing a logo, which is used in the Rancher UI. Could you please add a logo by setting the `icon` key as following if you have one hosted somewhere?
`icon: https://url-to-your-logo.png`

Alternatively, you can use this repository to host it by dropping the file in `/partner-charts/assets/logos/curiefense.png` and then referencing it here as `https://partner-charts.rancher.io/assets/logos/curiefense.png`.

**Review by PennyScissors — CHANGES_REQUESTED** [review]
Please test this chart within Rancher. I was able to deploy it, but not resources are created.
image

File: packages/curiefense/generated-changes/patch/Chart.yaml.patch [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+ name: curiefense
+-version: 1.0.0
+\ No newline at end of file
++version: 1.0.0
++
++kubeVersion: ">= 1.19"
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: curiefense
++  catalog.cattle.io/display-name: curiefense
```
Comment (PennyScissors) [inline packages/curiefense/generated-changes/patch/Chart.yaml.patch:15]: This specific annotation defines the name the UI should display and should be capitalized.
`catalog.cattle.io/display-name: Curiefense`

Comment (sergiopsyalo) [reply packages/curiefense/generated-changes/patch/Chart.yaml.patch:15]: We have to retake this. Let us see this weekend.

Comment (jdorfman) [reply packages/curiefense/generated-changes/patch/Chart.yaml.patch:15]: @sergiopsyalo @JuanPabloGarciaMonzon any updates?

**Comment by PennyScissors** [comment]
Closing this as there hasn't been any activity for a while and we recently updated our packaging scripts so it will likely be easier to submit a new PR to avoid rebasing issues.

**Comment by JuanPabloGarciaMonzon** [comment]
Yeah Iam working in a new PR with the updates of the guide.

**Pull request closed without merging**